03_launch_mgmt_cluster.sh

```diff
@@ -98,7 +98,38 @@ function clone_repos() {
     popd
 }
 
+function build_local_images() {
```

Comment:
Not in this pr but it'd be nice to figure out a way to de-duplicate this code between dev-scripts and metal3-dev-env